### ISSUE TITLE
install_linux_deb failed

### ISSUE THREAD

**Issue by williamdemeo** [TypeFunc/uh-mfc#2]
I'll try to get to the bottom of why the install script is failing on my machine, but here are my specs, in case it helps:

Chipset: Intel® CPU U7300 @ 1.30GHz × 2 
Distro: Ubuntu 16.04 LTS

The log of my install:
[gloss-rendering-1.10.3.5.log](https://github.com/TypeFunc/uh-mfc/files/931426/gloss-rendering-1.10.3.5.txt)

**Comment by jfennick** [comment]
Hi,
Can you run these commands and report the output?

cd ~/uh-mfc/haskelltalk
stack install gloss-rendering

**Comment by williamdemeo** [comment]
Well, I'm home now and the laptop I tried this on is in my office.  But I'll try again on my other laptop and, this time, I'll look through the install script and cherry pick a subset of install steps that I think will work for my machine.  I'll report back once I get it working.

**Comment by williamdemeo** [comment]
I made some major modifications to the install_linux_deb script.  I'll create a pull request.  Jake, could you try out the script on your machine(s) and, if it seems to work, please accept/merge the pull request.  Thanks!

**Comment by jfennick** [comment]
Hi,
I just wiped two of my machines, reinstalled Ubuntu 16.04 LTS, ran my install script, and everything is working on both machines.  I'm not sure what caused your error, but can you cd to the haskelltalk directory, run "stack build", and verify that you have an issue?  I'll try your script and merge your pull request once I've tested it.

**Comment by williamdemeo** [comment]
I've identified what probably caused most, if not all, of my problems using the sctip: the path ~/uh-mfc/haskelltalk is hardcoded in the script and that's not the location of my clone of the repo.

Once that is changed, your script may work, but I updated the script so that it warns the user about what packages are to be installed and also gives the option of not overwriting their custom .emacs file.  The new version of the script works, including the "stack build" command.
